<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPartition</class>
 <widget class="QDialog" name="AddPartition">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Partition</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="mainLayout">
     <item row="2" column="0">
      <spacer name="hSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="Line" name="separator_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="buttonSavePartition">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QGridLayout" name="innerLayout">
       <item row="0" column="2" colspan="2">
        <spacer name="hSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="buttonChoosePartition">
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="partitionName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Location">
         <property name="text">
          <string>Location</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_Name">
         <property name="text">
          <string>Partition Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QPlainTextEdit" name="partitionLocation">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="buttonCancelPartition">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
